generators/server/templates/_README.md

```diff
@@ -1,14 +1,23 @@
 # <%= baseName %>
 
-This application was generated using JHipster, you can find documentation and help at [https://jhipster.github.io](https://jhipster.github.io).
+This application was generated using JHipster <%= jhipsterVersion %>, you can find documentation and help at [<%= DOCUMENTATION_ARCHIVE_URL %>](<%= DOCUMENTATION_ARCHIVE_URL %>).
+<%_ if (applicationType == 'gateway' || applicationType == 'microservice' || applicationType == 'uaa') { _%>
 
-<%_ if (applicationType == 'gateway') { _%>
-<%- include('_micro_services_architecture.md'); %>
+This is a "<%= applicationType %>" application intended to be part of a microservice architecture, please refer to the [Doing microservices with JHipster][] page of the documentation for more information.
+<% if (applicationType == 'uaa') { %>
+This is also a JHipster User Account and Authentication (UAA) Server, refer to [Using UAA for Microservice Security][] for details on how to secure JHipster microservices with OAuth2.<% } %>
 <%_ } _%>
+
 ## Development
 
+<%_ if(skipClient) { _%>
```

Comment:
Nope it is skipClient so that we show only server instructions. See the other !skipClient if that also contains info about gulp. But it was also confusing to me...